src/Compiler/CompilerUtils.cpp

```diff
@@ -4,7 +4,7 @@
 
 //===-------------------------- CompilerUtils.cpp -------------------------===//
 //
-// Copyright 2019-2021 The IBM Research Authors.
+// Copyright 2019-2022 The IBM Research Authors.
```

Comment:
can we say `2019-now`

Reply:
I do not think we can... AFAIK copyright needs to be updated when we change the content of the file.